Networking/Sources/Networking/Networking/DynamicCodingKeys.swift

```diff
@@ -0,0 +1,12 @@
+struct DynamicCodingKeys: CodingKey {
+    var intValue: Int?
+    init?(intValue: Int) {
+        self.intValue = intValue
+        self.stringValue = "\(intValue)"
+    }
+
+    var stringValue: String
+    init(stringValue: String) {
+        self.stringValue = stringValue
+    }
```

Comment:
```suggestion
    let intValue: Int?
    init?(intValue: Int) {
        self.intValue = intValue
        self.stringValue = "\(intValue)"
    }

    let stringValue: String
    init(stringValue: String) {
        self.stringValue = stringValue
    }
```

To avoid duplication, you might consider making it generic, or at least have two versions for both Int and String.